src/platform/forms-system/src/js/components/ContactInfo.jsx

```diff
@@ -178,15 +179,15 @@ const ContactInfo = ({
 
   useEffect(
     () => {
-      if (missingInfo.length) {
+      if ((hasInitialized && missingInfo.length) || testContinueAlert) {
```

Comment:
I forgot to include this `hasInitialized` check here... but the unit test wasn't updating properly, so I added this `testContinueAlert`